tests/drivers/counter/counter_basic_api/src/test_counter.c

```diff
@@ -342,7 +342,7 @@ void test_all_channels_instance(const char *str)
 	dev = device_get_binding(str);
 	ticks = counter_us_to_ticks(dev, COUNTER_PERIOD_US);
 
-	alarm_cfgs.absolute = false;
+	alarm_cfgs.flags = COUNTER_ALARM_FLAG_ABSOLUTE;
```

Comment:
I guess it should be `= 0` as `absolute` was `false` previously.

Reply:
right...